### PR TITLE
Update MindGeek.yml (add Noirmale.com)

### PR DESCRIPTION
**Pull request by baloodusudouest**

### CODE REVIEW

**Comment by DogmaDragon** [comment]
Please update the data on the last line `# Last Updated October 14, 2022`.

File: scrapers/MindGeek.yml [hunk 1]
```diff
@@ -36,6 +37,7 @@ sceneByURL:
       - transangels.com/scene/
       - transsensual.com/scene
       - trueamateurs.com/scene/
+      - noirmale.com/scene/
```
Comment (hphpanon) [inline scrapers/MindGeek.yml:40]:
```suggestion
```

Per the automated PR check validation error:

```
URLs for type `scene` should be unique, already exists on /sceneByURL/0/url/15
```

I'm not clear what the value of having two separate lists of URLs for the same scraper type (or if there even is one), but you don't want the same URL listed twice.

Comment (nrg101) [reply scrapers/MindGeek.yml:40]: the two separate lists are for `sceneScraper` and `scriptScraper`, where presumably the domains work with one or the other

**Comment by bnkai** [comment]
Added to scriptscraper section and merging for now.
The preferred scraper should be the new MindGeekApi one, will update the relevant PR(#1240) once the scraper is ok.

**Pull request merged**